### ISSUE TITLE
Error with converting cert to Certificate model

### ISSUE THREAD

**Issue by rrroyal** [kapi2289/vulcan-api#25]
`cert = open("cert.json").read()`

Error when using `Vulcan(cert).get_grades()` with certificate as `str`:
```
Traceback (most recent call last):
  File "xyz", line 34, in <module>
    print(Vulcan(cert).get_grades())
  File "xyz\vulcan\_vulcan.py", line 25, in __init__    self._api = Api(certificate)
  File "xyz\vulcan\_api.py", line 14, in __init__   
    self.cert = to_model(Certificate, certificate)
  File "xyz\related\functions.py", line 107, in to_model
    value = cls(value)
TypeError: __init__() missing 3 required positional arguments: 'key', 'key_formatted', and 'base_url'
```

When I'm using `Vulcan(json.loads(cert))` there's no error, but there's no output too:
```python
print("[*] print(Vulcan(json.loads(cert)).get_grades())")
print(Vulcan(json.loads(cert)).get_grades())

print("[*] Vulcan(json.loads(cert)).get_grades()")
Vulcan(json.loads(cert)).get_grades()
```
Output:
```
[*] print(Vulcan(json.loads(cert)).get_grades())
<generator object Grade.get at 0x000001B9416BDF48>
[*] Vulcan(json.loads(cert)).get_grades()
```

**Comment by kapi2289** [comment]
You must pass the certificate as json. Please read about generator objects in Python.

Tl;dr try this:
```python
v = Vulcan(cert)

grades = list(v.get_grades())
print(grades)

# or you can iterate over generator
for grade in v.get_grades():
    print(grade)
```
